ci/jenkins/jobs/projects.yaml

```diff
@@ -634,3 +634,45 @@
               send-to-individuals: false
           triggers: []
           wrappers: []
+      - '{name}-{test_name}-matrix-test':
```

Comment:
can we add a `git-parameter` parameter so that a specific antrea ref can be tested on demand? @lzhecheng 

or are we handling this differently?

Reply:
No, it hasn't been done. I think the next release is close so we should focus on merging the scheduled job first? @ZhangYW18 what's your opinion? The plan is to be able to manually trigger matrix job with parameter like branch name, ref as well.

Reply:
Added a parameter called GIT_BRANCH. As this parameter can match multiple patterns, I added a detailed description of it from the document of Jenkins.